data/Dels/Dels missions.txt

```diff
@@ -925,6 +927,44 @@ mission "Dels Officer: Escort Convoy"
 	on visit
 		dialog phrase "generic arrived-without-npc dialog"
 
+mission "Dels Officer: Bad Officer 1"
+	repeat
+	landing
+	to offer
+		"dels officer: bad officer" >= 10
+	on offer
+		fine 10000
+		"dels officer: demerit count" += "dels officer: bad officer"
+		"dels officer: bad officer" = 0
+		dialog `Suddenly, you recieve a message from navy command! It reads, "Officer <first>, we are highly disappointed in you, as you have received many demerits. Because of this, you are being fined 10,000 credits."`
+
+mission "Dels Officer: Bad Officer 2"
+	landing

```

Comment:
Might need `invisible` tag so the name don't show up in job board.

Reply:
Done, thanks!